src/test/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtOpeningTimeServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.dts.fact.services.admin;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.entity.Court;
+import uk.gov.hmcts.dts.fact.entity.CourtOpeningTime;
+import uk.gov.hmcts.dts.fact.model.OpeningTime;
+import uk.gov.hmcts.dts.fact.repositories.CourtRepository;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ExtendWith({SpringExtension.class, MockitoExtension.class})
+@ContextConfiguration(classes = AdminCourtOpeningTimeService.class)
+public class AdminCourtOpeningTimeServiceTest {
+    private static final String COURT_SLUG = "some slug";
+    private static final String TEST_TYPE1 = "testType1";
+    private static final String TEST_TYPE2 = "testType2";
+    private static final String TEST_TYPE3 = "testType3";
+    private static final String TEST_HOURS = "testHours";
+
+    private static final int OPENING_TIME_COUNT = 3;
+    private static final List<CourtOpeningTime> COURT_OPENING_TIMES = new ArrayList<>();
```

Comment:
Think you mentioned using: 

`private static final List<CourtOpeningTime> COURT_OPENING_TIMES = emptyList()` on the other PR

Reply:
Same comment as above :)